src/panels/SkyBrowserPanel/WorldWideTelescope/WorldWideTelescopeView.tsx

```diff
@@ -14,20 +15,56 @@ import {
   useUpdateOpacities,
   useUpdateSelectedImages
 } from './hooks';
+import { Overlay } from './Overlay';
+
+import styles from './WorldWideTelescope.module.css';
 
 export function WorldWideTelescopeView() {
+  const cameraInSolarSystem = useAppSelector(
+    (state) => state.skybrowser.cameraInSolarSystem
+  );
+  const nBrowsers = useAppSelector((state) => state.skybrowser.browserIds.length);
+  const id = useAppSelector((state) => state.skybrowser.selectedBrowserId);
+
   const [isDragging, setIsDragging] = useState(false);
   const [startDragPosition, setStartDragPosition] = useState({ x: 0, y: 0 });
 
-  const { ref } = useWwtProvider();
-
+  const { ref, imageCollectionLoaded } = useWwtProvider();
   const { width, height } = useWindowSize();
   const luaApi = useOpenSpaceApi();
 
-  const nBrowsers = useAppSelector((state) => state.skybrowser.browserIds.length);
-  const id = useAppSelector((state) => state.skybrowser.selectedBrowserId);
+  const bind = useGesture(
+    {
+      onDrag: (state) => {
+        state.event.preventDefault();
+        if (!state.wheeling && !state.pinching && state.touches === 1) {
+          handleDrag(state.offset[0], state.offset[1]);
+        }
+      },
+      onDragStart: (state) => {
+        if (state.touches === 1 && !state.wheeling && !state.pinching) {
+          mouseDown(state.offset[0], state.offset[1]);
```

Comment:
I assume that the state.offset is some kind of delta change in the x and y direction? Do we want to give these names or write a short comment about them? If they are documented in the IDE it's fine (I haven't checked)